sdk-extensions/autoconfigure-spi/src/main/java/io/opentelemetry/sdk/autoconfigure/spi/ConfigProperties.java

```diff
@@ -93,12 +160,46 @@ public interface ConfigProperties {
   List<String> getList(String name);
 
   /**
-   * Returns a map-valued configuration property. The format of the original value must be
-   * comma-separated for each key, with an '=' separating the key and value. For instance, <code>
+   * Returns a List value configuration property.
+   *
+   * @see ConfigProperties#getList(String name)
+   * @return a List configuration property or {@code defaultValue} if a property with {@code name}
+   *     has not been configured.
+   * @throws ConfigurationException if the property is not a valid string.
+   */
+  default List<String> getList(String name, List<String> defaultValue) {
+    List<String> value = getList(name);
+    return (value == null || value.isEmpty()) ? defaultValue : value;
```

Comment:
`getList` and `getMap` can't return `null`

Reply:
Good point, I'll remove the null checks.